lib/lib-javadoc/src/main/resources/SQLSpecification2.csv

```diff
@@ -65,3 +65,4 @@ Reserved Columns,"In this example we are listing all rows that have a current ve
 Operations on LIST columns,"To filter on a list column. Use the HAS() clause, which holds a comma separated list of values. The values in the comma separated list are in a OR relationship meaning that a row matches if it contains any of the values.","select * from syn123 where fooList has ('either this value', 'or this value')"
 Operations on LIST columns,To filter for lists that contain must contain multiple values you will need chain together multiple HAS() clauses with ANDs. Order DOES NOT matter,select * from syn123 where fooList has ('first required value') and fooList has ('second required value') and fooList has ('third required value')
 Operations on LIST columns,"To operate on individual values stored inside a LIST column, use the UNNEST() function. NOTE: This will duplicate rows for all other columns.","select unnest(fooList), count(*) from syn123 group by unnest(fooList)"
+Other Synapse Functions, "Returns the Synapse userId of the current user.","select * from syn123 where userId = CURRENT_USER()"

```

Comment:
You can add a note that the CURRENT_USER() can be used in the select clause as well and the return column type is USERID